src/index.html

```diff
@@ -11,7 +11,186 @@
+              autocomplete="off"
+            >
+          </label>
+        </div>
+        <div class="input_field">
+          <label>
+            Full date of birth:
+            <input 
+              type="date"
+              name="birth_date"
+            >
+          </label>
+        </div>
+        <div>
+          <label>
+            I accept the term of the agreement 
+            <input type="checkbox">
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="group_field">
+        <legend>Registration</legend>
+        <div class="input_field">
+          <label>
+            E-mail:
+            <input 
+              type="email"
+              name="email"
+              placeholder="email@example.com"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+        <div>
+          <label>
+            Password:
+            <input 
+              type="password"
+              name="password"
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="group_field">  
+        <legend>An interesting fact about you!</legend>
+      
+        <div class="input_field">
+          Do you love cats?
+        
+          <label>
+            <input type="radio"
+              name="cats"
+              value="yep"
+            >
+          Yes
+          </label>
+          <label>
+            <input type="radio"
+            name="cats"
+            value="nope"
+            >
+          No
+          </label>
+        </div>
+
+        <div class="input_field">
+          <label>
+            What is your favorite color?
+            <input type="color"
+            name="color"
+            >
+          </label>
+        </div>
+
+        <div class="input_field">
+          <label>
+            What time do you go to bed?
+            <input 
+              type="time"
+              name="time"
+              step="2"
+            >
+          </label>
+        </div>
+
+        <div class="input_field">
+          <label class="form_field">
+            What are your favorite brand of cars?
+            <select name="car" 
+            required 
+            multiple
+            >
+              <option value="BMV">BMW</option>
+              <option value="Audi">Audi</option>
+              <option value="Lada">Lada</option>
+            </select>
+          </label>
+        </div>
+
+        <div>
+          <label>
+            How do you rate our work?
+            <input type="range"
+              value="0"
+              name="work"
+            >
+          </label>
+        </div>
+      </fieldset>
+    
+      <fieldset class="group_field">
+        <legend>Additional info:</legend>
+
+        <div class="input_field">
+          <label>
+            Comments: 
+            <textarea 
+              name="comments"
+              cols="20" 
+              rows="2"
+            >
+            </textarea>
```

Comment:
check this checklist point
<img width="961" alt="image" src="https://user-images.githubusercontent.com/102411130/226286085-0093f5b3-324e-4fa1-825b-51fbfee60f31.png">

it has redundant space now
<img width="302" alt="image" src="https://user-images.githubusercontent.com/102411130/226286193-3961faff-a406-4393-87fc-08f198bd4201.png">